cmd/list/gates/cmd.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gates
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/logging"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/output"
+	rprtr "github.com/openshift/rosa/pkg/reporter"
+)
+
+var args struct {
+	version string
+}
+
+var Cmd = &cobra.Command{
+	Use:     "gates",
+	Aliases: []string{"gates"},
+	Short:   "List available OCP Gates",
+	Long:    "List available OCP Gates.",
+	Example: `  # List all available OCP Gates 
+  rosa list gates --version 4.9`,
```

Comment:
I think it makes sense to allow a `--cluster` flag and show whether the ACKs have been set for its gates.

Reply:
Done